tool/zjit_bisect.rb

```diff
@@ -73,6 +73,25 @@ def run_bisect(command, items)
 end
 
 def run_ruby *cmd
+  if RUBY == "make"
+    # Automatically detect that we're running a make command instead of a Ruby
+    # one. Pass the bisection options via RUN_OPTS instead.
+    zjit_opts = cmd.select { |arg| arg.start_with?("--zjit") }
+    run_opts_index = cmd.find_index { |arg| arg.start_with?("RUN_OPTS=") }
+    specopts_index = cmd.find_index { |arg| arg.start_with?("SPECOPTS=") }
+    if run_opts_index
+      run_opts = Shellwords.split(cmd[run_opts_index].sub("RUN_OPTS=", ""))
+      run_opts.concat(zjit_opts)
+      cmd[run_opts_index] = "RUN_OPTS=#{run_opts.shelljoin}"
+    elsif specopts_index
+      specopts = Shellwords.split(cmd[specopts_index].sub("SPECOPTS=", ""))
```

Comment:
not sure if you like it, but there's API for doing it

```suggestion
      run_opts = Shellwords.split(cmd[run_opts_index].delete_prefix("RUN_OPTS="))
      run_opts.concat(zjit_opts)
      cmd[run_opts_index] = "RUN_OPTS=#{run_opts.shelljoin}"
    elsif specopts_index
      specopts = Shellwords.split(cmd[specopts_index].delete_prefix("SPECOPTS="))
```